src/components/Calculator/ScoreTable.js

```diff
@@ -18,33 +30,65 @@ const ScoreTable = ({ subjects }) => (
             Kierunek
           </th>
           {/* <th scope='row'>
-            Przedmioty
-          </th> */}
+              Przedmioty
+            </th> */}
           <th scope='row'>
             Wynik
           </th>
+          <th>
+            2013
```

Comment:
(minor) What about getting available years from `thresholds.json` and rendering them?
That way your view will automatically update when you add another year values to the file.